trinity/db/chaindb.py

```diff
@@ -0,0 +1,62 @@
+from evm.db.chain import (
+    BaseChainDB,
+)
+
+from trinity.utils.ipc import (
+    ObjectOverIPC,
+    IPCMethod,
+)
+
+
+class PipeChainDB(ObjectOverIPC, BaseChainDB):
```

Comment:
I almost went this far but I'm planning to iterate towards that in the next release and then probably pull it out into a stand-alone library since my quick googling didn't yield any results.